src/proto/FunctionRpc.proto

```diff
@@ -648,3 +650,11 @@ message RpcHttp {
   map<string,NullableString> nullable_params = 21;
   map<string,NullableString> nullable_query = 22;
 }
+
+message BindingData
+{
+    string Version = 1;
```

Comment:
Just a note that I think the convention in this file is lowercase properties. Don't think this will break anything though. 

thx for linking design docs and context in the description will take a look at that before reviewing more :)

Reply:
fair point, will make lowercase!